README.md

```diff
@@ -37,11 +37,18 @@ requiring an upgrade to our current version.
 
 ### Installing the easy way
 
-1) Run the setup script (MacOS, Linux)
-
+1) Run the setup script 
+   
+    
+   MacOS / Linux:
    ```
    $ ./tools/setup
    ```
+   
+   Windows (Powershell): 
+   ```
+   $ ./tools/setup.ps1  
```

Comment:
If we were retro we'd use '\' rather than '/' here... I think both work now, but, like, it permutes the latter into the former.

Reply:
Was supposed to be a back-slash above.